modules/ROOT/pages/sdk-xattr-example.adoc

```diff
@@ -186,3 +186,62 @@ public class CouchbaseExtendedAttributesDemo
     }
 }
 ----
+
+== Virtual Extended Attributes Example
+
+Using the subdocument API, xref:sdk-xattr-overview.adoc#_virtual_extended_attributes[Virtual XATTR] can be used to fetch metadata about a document, via the `$document` virtual XATTR.
+A common use case is discovering documentation expiry data, or TTL:
+
+[source,java]
+----
+import java.text.SimpleDateFormat;
+import java.util.Date;
+ 
+import com.couchbase.client.core.message.kv.subdoc.multi.Lookup;
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.Cluster;
+import com.couchbase.client.java.CouchbaseCluster;
+import com.couchbase.client.java.subdoc.*;
+ 
+public class GetTTL
+{
+    public static void main(String [] args)
+    {
+        // Access the cluster that is running on the local host, authenticating with
+        // the username and password of the Full Administrator. This
+        // provides all privileges.
+        //
+        Cluster cluster = CouchbaseCluster.create("localhost");
+        cluster.authenticate("test", "password");
+        Bucket test = cluster.openBucket("test");
+ 
+        String key = "Perry";
```

Comment:
better to change this and the lines above to something out of travel-sample, the canonical example?

I usually use the entry for United Airlines (since they're a user)